pkg/controller.v2/controller_status.go

```diff
@@ -64,7 +64,7 @@ func (tc *TFJobController) updateStatus(tfjob *tfv1alpha2.TFJob, rtype tfv1alpha
 			msg := fmt.Sprintf("TFJob %s is successfully completed.", tfjob.Name)
```

Comment:
Does the condition for determining success need to be updated? Now that you support chief per your other PRs? I think I saw code in your other PR to handle this so maybe you just need to sync?

Reply:
Yeah, this PR has conflicts with other PRs, and I will rebase after other PRs are all merged.